test/CMakeLists.txt

```diff
@@ -11,7 +11,9 @@ FIND_LIBRARY(MESSAGEMANAGER messagemanager ${CMAKE_CURRENT_SOURCE_DIR}/../includ
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../include)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../src/proto3)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../src/grpc)
+include_directories(/home/ubuntu/src/github.com/zzxgzgz/marl/include)
```

Comment:
we'll figure what path to put, need to make corresponding changes in machine-init.sh

Reply:
fixed aca-machine-init.sh with cmake